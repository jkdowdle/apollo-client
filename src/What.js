import React from 'react'
import { ApolloProvider } from 'react-apollo'
import { Provider } from 'react-redux'

// import styles from './index.css'

// import App from './components/App';
import App from './redux/components/App'
import client from './app-client';
import store from './app-store'

export const What = () => (
  <Provider store={store}>
    <ApolloProvider client={client}>
      <App />
    </ApolloProvider>
  </Provider>
)

export default What