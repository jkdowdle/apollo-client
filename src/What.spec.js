import React from 'react'
import { render } from 'enzyme'
import toJson from 'enzyme-to-json'

import What from './What'

it('works', () => {
  const tree = render(<What />)

  

  expect(toJson(tree)).toMatchSnapshot()
})