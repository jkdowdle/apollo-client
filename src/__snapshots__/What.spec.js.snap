// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works 1`] = `
<div>
  <ul
    style="display:flex;justify-content:space-evenly"
  >
    <li>
      <a
        href="/"
      >
        Users
      </a>
    </li>
    <li>
      <a
        href="/register"
      >
        Register
      </a>
    </li>
    <li>
      <a
        href="/profile"
      >
        Profile
      </a>
    </li>
  </ul>
  <div>
    <h1>
      Users
    </h1>
    <div>
      <h3>
        Search Users
      </h3>
      <input
        name="searchTerm"
        type="text"
        value=""
      >
        
      </input>
       Age:
      <select
        name="orderBy"
      >
        <option
          value=""
        >
          Default
        </option>
        <option
          value="desc"
        >
          Highest
        </option>
        <option
          value="asc"
        >
          Lowest
        </option>
      </select>
    </div>
    Loading . . .
  </div>
</div>
`;
